datacenter/ucp/2.1/guides/admin/configure/content-trust/client_configuration.md

```diff
@@ -0,0 +1,255 @@
+description:
+---
+
+After an administrator
+[configures the UCP and DTR servers for content trust](admin_tasks.md) and
+[delegates users to be able to sign images](admin_tasks.md#delegate-image-signing),
+each of those users needs to configure their system for image signing. This
+topic covers the required configuration steps, as well as how to sign and
+push images.
+
+Each user who will sign images needs to follow these steps.
+
+## Import the user's signing key
+
+Users who need to sign images should import the `key.pem` file from their UCP
+client bundle into Docker. The user probably downloaded the client bundle when
+they sent the administrator their `cert.pem` so the administrator could
+[delegate signing to them](#delegate-image-signing).
+
+Import the private key associated with the user certificate. You must specify
+the trust directory `~/.docker/private`. If the `~/.docker/private` directory
+does not yet exist, Notary will create it.
+
+- **Linux or macOS**:
+
+  ```bash
+  $ notary -d ~/.docker/trust key import /path/to/key.pem
+  ```
+
+- **Windows**:
+
+  ```powershell
+  PS C:\> notary -d ~/.docker/trust key import /path/to/key.pem
+  ```
+
+You are prompted for a passphrase. Save it in a secure location such as a
+password manager. You will need to provide the passphrase each time you sign an
+image.
+
+## Configure the Docker client
+
+These steps may need to be performed for each Docker user who will push images
+to the trusted repository, and also on each Docker client which should only be
+allowed to pull and use trusted images.
+
+### Linux or MacOS
+
+1.  **Required**: Set the `DOCKER_CONTENT_TRUST` environment variable to `1`.
+    You can do this temporarily or permanently.
+
+    - To set the environment variable for the current command-line session, type
+      the following into the terminal where you will be running `docker` commands:
+
+      ```bash
+      $ export DOCKER_CONTENT_TRUST=1
+      ```
+
+      This environment variable will be effective until you close the command
+      prompt.
+
+    - To set the environmment variable for just a single command, add it before
+      the command:
+
+      ```bash
+      $ DOCKER_CONTENT_TRUST=1 docker pull...
+      ```
+
+    - To set the environment variable permanently, edit the `~/.profile` file
+      and add the following line:
+
+      ```bash
+      export DOCKER_CONTENT_TRUST=1
+      ```
+
+2.  **If your DTR instance uses certificates not signed by a public certificate authority (CA)**:
+    Configure the local Docker daemon and client to trust the DTR server's
+    certificate. You need to do this step if you see an error like the following
+    when you try to [sign and push an image](#sign-and-push-an-image).
+
+    ```none
+    x509: certificate signed by unknown authority
+    ```
+
+    This procedure is different if you are on a Linux or macOS client:
+
+    - **Linux**:
+
+      1.  Download the certificate add it to a subdirectory of the
```

Comment:
Yup, we should not be storing CAs in the `/etc/docker/certs.d`, instead they should configure the OS for this. This has the added benefit that any other client tools will trust the CA.
E.g. If IT decides to rollout DDC with their own PKI, they can configure employee laptops to trust the CA once, and Docker, compose, curl, Chrome, Firefox and others will trust that CA.

We already have docs on [how to achieve this](https://docs.docker.com/datacenter/dtr/2.2/guides/user/access-dtr/)